### DISCUSSION TITLE
Cannot deploy Hello World

### DISCUSSION THREAD

**Discussion by soumilshah1995** [logandk/serverless-wsgi#231] [Q&A] [answered]
Error:
'Unable to load virtualenv, please install\r\n'
× Stack example-dev failed to deploy (0s)
Environment: win32, node 16.18.0, framework 3.23.0, plugin 6.2.2, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
'Unable to load virtualenv, please install\r\n'

**Comment by keeganstothert** [comment]
hey did you ever solve this @soumilshah1995 ?

**Comment by soumilshah1995** [comment]
No I didn’t !!
On Sun, Dec 4, 2022 at 3:45 PM Keegan Stothert ***@***.***>
wrote:

> hey did you ever solve this @soumilshah1995
> <https://github.com/soumilshah1995> ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/logandk/serverless-wsgi/discussions/231#discussioncomment-4308028>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJMF5PYP5LGQKACQFXXPRIDWLT7HXANCNFSM6AAAAAASTSAYVU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Thanking You,
Soumil Nitin Shah

**Reply by keeganstothert** [reply]
I switched to `serverless-python-requirements` to handle requirements and the issues seem to be resolved

**Comment by satyamjay-iitd** [comment, marked as answer]
Just add packRequirements: false. Add packages manually or use serverless-python-requirements to package. 'packRequirements' seems to be broken for later python versions.

```
custom:
  wsgi:
    app: api.app
    packRequirements: false
```
